src/user-selection/components/users.css

```diff
@@ -41,6 +50,12 @@
   border-right: dashed 1px black;
 }
 
+.removeUser {
+  display: none;
+  position: relative;
```

Comment:
I would use absolute positioning here, so it doesn't break the centred styling of the text.